fashion-example-query/README.md

```diff
@@ -5,81 +5,101 @@
+There are some requirements to run this example, to download everything you need, just run in your terminal:
 
 ```
 pip install -r requirements.txt
 ```
 
-
 ## 🏃 Run the Flows
 
-Now that we've got the code to load our data, we're going to dive into writing our app and running our Flows!
+We usually need at least two Flows in Jina. One for Indexing and one for Querying. 
 
 ### Index Flow
 
-To run the index you type:
-
-```bash
-python app.py -t index
-```
-First up we need to build up an index and then we'll search through this index when we use the query Flow later.
-So what is happening here?
+First thing we need is to index our data, and we need a Flow to manage this. 
 
-There are 10 categories in the fashion-mnist-data, but to simplify this example we will only use 3 categories:
+ ```python
+def index(num_doc, target: dict):
+    f = Flow.load_config('flows/index.yml')
+    with f:
+        f.index(index_generator(num_doc, target), request_size=2048)
+ ```
+      
+There are 10 categories in the fashion-mnist-data, but to simplify this example we will only use 3 of them:
 
     0	        T-shirt/top
     1	        Trouser
     2	        Pullover
-    
-We will create a separate index per category, so each category will have its own yml file and then we index the data updating updating the label
-
-```
+ 
+ We will create a separate index per category, so each category will have its own Yaml file. We can then index the data updating its label
+ 
+ ```python
 d.tags.update({'label': get_mapped_label(label_int)})
 ```
-This is where you could tweak the code if you would like to see only one category.
-For example if you would like to see only pullovers, you could do something like this
+ 
+This is where you could tweak the code if you would like to see only one category. For example if you would like to see only pullovers, you could do something like this
+ 
+ ![alt text](.github/images/filter.png "Pullover filter")
+ 
+ Then we have ready all the indexes!
 
-![alt text](.github/images/filter.png "Results")
 
-Then we have ready all the indexes!
+You can run all this with the command:
 
+```bash
+python app.py -t index
+```
 
 ### Query Flow
 
-Now for the query time, run:
+The query Flow is similar but it uses the  `query.yml ` file instead of `index.yml`
+
+ ```python
+def query(num_doc, target: dict):
+    f = Flow.load_config('flows/query.yml')
+    with f:
+        f.search(query_generator(num_doc, target), shuffle=True, size=128,
+                 on_done=print_result, request_size=32, top_k=TOP_K)
+    write_html(os.path.join('./workspace', 'hello-world.html'))
+ ```
+
+To query just run:
 
 ```bash
 python app.py -t query
 ```
 
-## Results
+## 🌟 Results
 
-So if we run this as it is, we will see our results filtered by pullovers like this:
+If you run this code as it is, you will see the results filtered by pullovers like this:
 
 ![alt text](.github/images/results.png "Results")
 
 
-## QueryLanguage
+## 🧞 QueryLanguage
```

Comment:
Should we link to the doc or blog post by Han on query lang as next steps?

Reply:
Yes you are right